guestbook-go/go.mod

```diff
@@ -0,0 +1,14 @@
+module Guestbook
+
+go 1.20
```

Comment:
Here you're using go 1.20 and the image is 1.18, that doesn't seem to match, why? Also in https://github.com/kubernetes/examples/pull/542 I see go 1.22 used, why not going all the way, since that seems to work?